cmd/notation/main.go

```diff
@@ -30,4 +34,30 @@ func main() {
 	if err := cmd.Execute(); err != nil {
 		os.Exit(1)
 	}
+
+	// generates MD CLI docs if the GEN_CLI_DOCS env var is "true"
+	genDocs := os.Getenv("GEN_CLI_DOCS")
```

Comment:
Instead of doing it here should we create a new command to generate markdowns like [clockify](https://github.com/daachi/clockify-cli/blob/77d97d9537fcb311735989bed62fe969995dc3e3/cmd/gendocs.go#L39-L86]? Its much cleaner and allows you to generate markdown files to given destination

Reply:
@priteshbandi I'm open to discussing this approach, but I'd also like @FeynmanZhou  and @yizha1 to weigh in if they have a preference.

Reply:
thank you @priteshbandi 

I've refactored to incorporate the approach you mentioned.